Misc/NEWS.d/next/Library/2021-01-31-18-24-54.bpo-43086.2_P-SH.rst

```diff
@@ -0,0 +1 @@
+Added handling for excess data in binascii.a2b_base64
```

Comment:
mention the new strict_mode parameter.  Also update Doc/library/binascii.rst to mention it, including an appropriate versionadded marker.

Reply:
Done in 464484c.